src/System.Net.Primitives/tests/FunctionalTests/CredentialCacheTest.cs

```diff
@@ -392,24 +431,101 @@ public static void DefaultCredentials_Get_Success()
             Assert.Equal(String.Empty, c.Password);
             Assert.Equal(String.Empty, c.Domain);
         }
-
-        [Fact]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.Netcoreapp | TargetFrameworkMonikers.Uap)]
-        public static void AddRemove_UriAuthenticationTypeDefaultCredentials_Success_net46()
+        
+        [Theory]
+        [MemberData(nameof(StandardAuthTypeWithNetworkCredential))]
+        [MemberData(nameof(CustomAuthTypeWithCustomNetworkCredential))]
+        public static void Add_UriAuthenticationType_Success(string authType, NetworkCredential nc)
         {
-            NetworkCredential nc = CredentialCache.DefaultNetworkCredentials as NetworkCredential;
+            // Default credentials cannot be supplied for the Basic authentication scheme.
+            if (string.Equals(authType, authenticationTypeBasic, StringComparison.OrdinalIgnoreCase) && (nc == CredentialCache.DefaultNetworkCredentials))
+            {
+                return;
+            }
+            
             CredentialCache cc = new CredentialCache();
-
-            // The full framework throw System.ArgumentException with the message
-            // 'Default credentials cannot be supplied for the authenticationType1 authentication scheme.'
-            AssertExtensions.Throws<ArgumentException>("authType", () => cc.Add(uriPrefix1, authenticationType1, nc));
+            
+            // .Net Framework and .Net Core have different behaviors for Digest when default NetworkCredential is used.
+            if (string.Equals(authType, authenticationTypeDigest, StringComparison.OrdinalIgnoreCase) && (nc == CredentialCache.DefaultNetworkCredentials))
+            {
+                if (PlatformDetection.IsFullFramework)
+                {
+                    // In .Net framework, when authType == Digest, if WDigestAvailable == true, it will pass the validation.
+                    // if WDigestAvailable == false, it will throw ArgumentException.
+                    // In order to determine WDigestAvailable's value, we need to use a method in SSPIWrapper.cs
```

Comment:
I don't think it's useful to talk about the SSPIWrapper.cs problem here.  Just state "that it is not possible to easily determine if Digest is supported or not on .NET Framework.  So, we will skip the test"